soul-common/src/main/java/org/dromara/soul/common/enums/PluginEnum.java

```diff
@@ -82,6 +82,11 @@
      */
     DIVIDE(50, 0, "divide"),
 
+    /**
+     * WebClient plugin enum.
+     */
+    WEB_CLIENT(51, 0, "webClient"),
```

Comment:
why and this??

Reply:
to compare with webClientPlugin.getOrder() and webClientPlugin.named()

Reply:
don't modify here for test

Reply:
oh, I will change back